docs/evaluation/combine.rst

```diff
@@ -441,24 +493,134 @@ manner to models::
     >>> g2.pos
     <Parameter 'pos' of model 'g2'>
     >>> print(g2.pos)
-    val         = 0.504315379302
-    min         = -3.40282346639e+38
-    max         = 3.40282346639e+38
-    units       = 
+    val         = 0.5001684477780305
```

Comment:
We run doctestplus with `doctest_plus_atol=1e-4` meaning that string output numbers are allowed to differ at the level of 1e-4 (for rtol we use the default, I think it's 1e-6). 
Of course, it's good to have the full precision as shown to the user on a screen, and I'm not requesting changes, I'm just saying if you want to display fewer digits similar to the previous version it wouldn't cause problems.